cmd/ringbuf/src/lib.rs

```diff
@@ -119,6 +116,8 @@ fn taskname<'a>(
     Ok(&hubris.lookup_module(HubrisTask::from(variable.goff))?.name)
 }
 
+// this allow is meant for line 151 but it does not work, so we do it here
```

Comment:
Maybe change this to indicate that it's a `println!` in the body rather than the specific line?  This comment will become *really* confusing if the file gets reordered. ;)